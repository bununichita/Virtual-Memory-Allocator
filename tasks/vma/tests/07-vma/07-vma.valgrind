==49276== Memcheck, a memory error detector
==49276== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==49276== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==49276== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==49276== Parent PID: 49275
==49276== 
==49276== 
==49276== HEAP SUMMARY:
==49276==     in use at exit: 0 bytes in 0 blocks
==49276==   total heap usage: 10 allocs, 10 frees, 8,377 bytes allocated
==49276== 
==49276== All heap blocks were freed -- no leaks are possible
==49276== 
==49276== For lists of detected and suppressed errors, rerun with: -s
==49276== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
