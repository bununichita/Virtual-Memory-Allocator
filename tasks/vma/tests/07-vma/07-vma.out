The allocated address is outside the size of arena
The allocated address is outside the size of arena
Total memory: 2 bytes
Free memory: 1 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 1 - 2
Miniblock 1:	1	-	2	| RW-
Block 1 end
1
1
